server/cluster/cluster.go

```diff
@@ -352,6 +341,37 @@ func (c *RaftCluster) Start(s Server) error {
 	return nil
 }
 
+func (c *RaftCluster) runServiceCheckJob() {
+	defer logutil.LogPanic()
+	defer c.wg.Done()
+
+	var once sync.Once
+
+	checkFn := func() {
+		if c.isAPIServiceMode {
+			once.Do(c.initSchedulers)
+			c.enabledServices.Store(mcsutils.SchedulingServiceName, true)
+		} else if !c.schedulingController.running.Load() {
+			c.startSchedulingJobs()
+			c.enabledServices.Delete(mcsutils.SchedulingServiceName)
```

Comment:
This `enabledServices` naming is confusing to me, it seems more like `servicesToEnable` since we will delete the service name from it after starting.

Reply:
If the scheduling server is started, the enabledServices will store it, otherwise, it will be deleted in enabledServices.

Reply:
It is also confusing to me

Reply:
Any better idea?

Reply:
How about `independentServices`?   And do we need maintain this in PD mode?